app/views/GermlineView/index.jsx

```diff
@@ -8,7 +8,7 @@ import {
 import './index.scss';
 
 const Board = lazy(() => import('./components/Board'));
-const Report = lazy(() => import('./components/Report'));
+const Report = lazy(() => import('./components/Report/index.tsx'));
```

Comment:
don't think this is needed?

Reply:
Leftover from when the angular components were there. I'll remove